chain.schema.json

```diff
@@ -2,14 +2,13 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "codebase":{
     "git_repo": "https://github.com/osmosis-labs/osmosis",
-    "recommended_version": "v15.0.0",
+    "recommended_version": "v15.1.0",
     "compatible_versions": [
+      "v15.1.0",
       "v15.0.0"
     ],
-    "binaries": {
```

Comment:
was the binaries title on this one and the one below removed intentionally?

Reply:
No, it was not intentional; good catch! and my apologies. Should be fixed now.